deep-security-agent/defaults/main.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright:: 2018, Trend Micro
```

Comment:
Why this copyright format is different from others? it seems :: should be removed?

Reply:
Sure. Then we should modify this file in Chef repo, too.
https://github.com/deep-security/chef/blob/master/deep-security-agent/attributes/default.rb